src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,65 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { actions as currentTodoActions } from '../../features/currentTodo';
 
 export const TodoModal: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const selectedTodo = useAppSelector(state => state.currentTodo);
+
+  const [user, setUser] = useState<User | null>();
+
+  useEffect(() => {
+    if (selectedTodo) {
+      getUser(selectedTodo.userId)
+        .then(setUser);
```

Comment:
Error handling is missing for the promise returned by 'getUser'. You should add a catch block to handle any errors that may occur.